sdk/.github/workflows/update_docs.yml

```diff
@@ -0,0 +1,84 @@
+name: Update Documentation
+
+on:
+  pull_request:
+    types: [closed]
+    branches:
+      - testnet3
+jobs:
+  build:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sdk
+        uses: actions/checkout@v3
+        with:
+          repository: 'AleoHQ/sdk'
+          path: 'sdk'
+
+      - name: Checkout aleo-docs
+        uses: actions/checkout@v3
+        with:
+          repository: 'AleoHQ/aleo-docs'
+          path: 'aleo-docs'
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Checkout welcome
+        uses: actions/checkout@v3
+        with:
+          repository: 'AleoHQ/welcome'
+          path: 'welcome'
+          token: ${{ secrets.GITHUB_TOKEN }}
+          submodules: recursive
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '20'
+
+      - name: Install dependencies
+        run: |
+          cd sdk
+          npm install
+          cd ../aleo-docs
+          yarn install
+
+      - name: Generate Docs
+        run: |
+          cd ../aleo-docs
+          yarn gen-sdk-docs
+
+      - name: Update submodule and Push
+        run: |
+          git config --global user.email "github-actions@github.com"
+          git config --global user.name "GitHub Actions"
+          cd ../welcome/sdk
+          git pull origin main
```

Comment:
Do changes in main end up auto-pushing to the developer website? 

I ask because if this is directly publishing to the website, we should take a look through the SDK docs and ensure they're correct (i.e. no spelling or logical errors) before merging this PR.

Also - do you have a link for how the docs currently look on the website?

Reply:
Yup that's right. Updates to jsdoc3 comments in the js sdk will be directly published to the website-- of course after the PRs created in welcome and aleo-docs are reviewed. This draft PR though won't be merged until we merge in this [PR](https://github.com/AleoHQ/aleo-docs/pull/223) and this [PR](https://github.com/AleoHQ/welcome/pull/244). 

I think reviewing the SDK docs to ensure correctness would be valuable on this front so I won't publish changes to the site until I get the green light from you and @onetrickwolf !

I don't have a preview link for how the docs currently look on the site-- running the branches mentioned earlier locally would be the best way to view current progress.

Reply:
Will mark this as resolved now, let me know if you have any other questions though!

Reply:
K, once we verify correctness, I'll send an approval for this PR

Reply:
Also @moAlobaidi feel free to go through the Javascript SDK and verify correctness as well. I think it should be straightforward enough to verify and if you find any errors, make a PR and we'll get the fixes integrated!